lib/shared/bucketing-assembly-script/assembly/types/configBody.ts

```diff
@@ -97,7 +99,15 @@ export class ConfigBody {
         this.features = features.valueOf().map<Feature>((feature) => {
             return new Feature(feature as JSON.Obj)
         })
-
+        const audiences = getJSONObjFromJSON(configJSONObj, 'audiences')
+        this.audiences = new Map<string, Audience>()
+        const audienceKeys = audiences.keys
+        for (let i=0; i < audienceKeys.length; i++) {
+            const audience_id = audienceKeys[i]
+            const aud = audiences.get(audience_id)
+            if (!aud) throw new Error(`Unable to get audience value for id: ${audience_id}`)
```

Comment:
how is this possible?

Reply:
I don't think it should be, I was just replicating what I saw in the variableHashes code. I can delete it

Reply:
Its been removed